x/stake/handler.go

```diff
@@ -114,8 +114,6 @@ func handleMsgCreateValidator(ctx sdk.Context, msg types.MsgCreateValidator, k k
 	}
 
 	k.OnValidatorCreated(ctx, validator.OperatorAddr)
-	accAddr := sdk.AccAddress(validator.OperatorAddr)
-	k.OnDelegationCreated(ctx, accAddr, validator.OperatorAddr)
```

Comment:
can't delete these, there is a delegation created (self delegation) which needs to be initialized

Reply:
it's now initialized with the `.Delegate()` call

Reply:
aha - yeah okay got it